### PR TITLE
Kernel_CheckTokenMiddleware

### PR DESCRIPTION
**Pull request by ifelseelif**
Add check token middleware in Kernel project.
Inject this middleware in all services middleware pipeline.
Exclude /login endpoint in AuthService

### CODE REVIEW

File: DigitalOffice/Shared/Kernel/TokenMiddleware/TokenMiddleware.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                }
+                catch (Exception e)
+                {
+                    context.Response.StatusCode = 500;
+                    await context.Response.WriteAsync(InternalMessageError);
+                }
+            }
+        }
+
+
```
Comment (KostinP) [inline DigitalOffice/Shared/Kernel/TokenMiddleware/TokenMiddleware.cs]: Extra empty lines

Comment (ifelseelif) [reply DigitalOffice/Shared/Kernel/TokenMiddleware/TokenMiddleware.cs]: fixed

**Review by DarkFelas — APPROVED** [review]

**Review by denis-kriv — APPROVED** [review]

**Review by KostinP — APPROVED** [review]

File: DigitalOffice/Services/CompanyService/Startup.cs [hunk 2]
```diff
@@ -95,6 +79,39 @@ private void UpdateDatabase(IApplicationBuilder app)
...
 
+        private void ConfigureRabbitMq(IServiceCollection services)
+        {
+            string appSettingSection = "ServiceInfo";
+            string serviceId = Configuration.GetSection(appSettingSection)["ID"];
+            string serviceName = Configuration.GetSection(appSettingSection)["Name"];
+
+            string appSettingMassTransitUris = "MassTransitUris";
+            string checkTokenUri = Configuration.GetSection(appSettingMassTransitUris)["CheckToken"];
+
```
Comment (mekk1t) [inline DigitalOffice/Services/CompanyService/Startup.cs:90]: This is dumb. You already hard-code "MassTransitUris", so why the extra string variable?
Either come up with a different configuration or hard-code it there as well.

**Review by mekk1t — APPROVED** [review]
Will there be any tests?

**Review by DarkFelas — APPROVED** [review]

**Review by lalagvana — APPROVED** [review]

**Pull request merged**